src/xcode/ENA/ENA/Source/HTTPClient/Service/Models/Receive/TestResultModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+// 🦠 Corona-Warn-App
+//
+struct TestResultModel: Codable {
+	let testResult: Int
+	let sc: Int?
+	let labId: String?
+
+	static func fake(
+		testResult: Int = 0,
+		sc: Int? = nil,
+		labId: String? = nil
+	) -> TestResultModel {
+		TestResultModel(
+			testResult: testResult,
+			sc: sc,
+			labId: labId
+		)
+	}
```

Comment:
Extend your `TestResultModel` in a new file and put the` fake()` func into that. At the end, assign the new file to the test target. So we ensure, the fake() func is only available in the test target.
Example: `TicketValidationAccessToken` and `TicketValidationAccessToken+Fake`

Reply:
Done